config.js

```diff
@@ -1,24 +1,24 @@
 module.exports = {
   issuer: {
     'silverlake-local': {
-      issuer: 'https://www.banno.com/a/consumer/api/oidc',
-      authorization_endpoint: 'https://silverlake.banno-production.com/a/consumer/api/oidc/auth',
-      token_endpoint: 'https://silverlake.banno-production.com/a/consumer/api/oidc/token',
-      jwks_uri: 'https://silverlake.banno-production.com/a/consumer/api/oidc/certs'
+      issuer: 'https://uat.banno.com/a/consumer/api/oidc',
+      authorization_endpoint: 'https://silverlake.banno-uat.com/a/consumer/api/oidc/auth',
+      token_endpoint: 'https://silverlake.banno-uat.com/a/consumer/api/oidc/token',
+      jwks_uri: 'https://silverlake.banno-uat.com/a/consumer/api/oidc/certs'
     }
   },
   client: {
     'silverlake-local': {
-      client_id: '42b799f2-e543-4910-b5ec-e69fd458814b', // These credentials are designed for *demonstration* purposes only.
-      client_secret: 'd75b6fb3-91b8-4935-9251-651b294249de', // These credentials are designed for *demonstration* purposes only.
+      client_id: '042bd8a9-f470-4804-baa2-4910df7de9cf', // These credentials are designed for *demonstration* purposes only.
+      client_secret: 'a85c2227-1198-4605-b2e5-326411aee27c', // These credentials are designed for *demonstration* purposes only.
```

Comment:
@ChadKillingsworth I'm not convinced that this branch is actually demonstrating a successful `PKCE` flow. While I do indeed see the `client callback` in the console with data that _looks_ promising (e.g. seeing the `code_verifier`), I only see 'successful' authentication if I have this `client_secret` in place.

_Hypothetically_, we shouldn't need to have the `client_secret` as part of this config if we're using the `PKCE` flow.  If I _remove_ the `client_secret` from the config, then I get a `401 - Unauthorized` error.

This behavior _roughly_ matches the behavior that I'm seeing when I run the AppAuth.io example using the above config information. I believe that the AppAuth.io example uses `PKCE`. I can get the AppAuth.io example app to let me log into Banno but the flow errors  with a `401 - Unauthorized` when attempting to make the token request.

Reply:
Resolving this concern as resolved by commit https://github.com/Banno/consumer-api-openid-connect-example/pull/11/commits/f270c52426c92b62bacf4b6b4724fce005f3832e.